scheduler/scheduler.go

```diff
@@ -66,24 +68,39 @@ func scheduleTask(interval time.Duration, initFunc func(), tickFunc func()) {
 }
 
 func checkMemberStatus() {
-	db, err := database.Setup()
-	if err != nil {
-		log.Errorf("error setting up db: %s", err)
-	}
-	defer db.Release()
+	db.ApplyMemberCredits()
+	db.UpdateMemberTiers()
 
-	db.EvaluateMembers()
-}
+	const memberGracePeriod = 46
+	const membershipMonth = 31
+
+	mailer := mail.NewMailer(db, mailApi, c)
+
+	pendingRevokation, err := db.GetCommunication(mail.PendingRevokationMember.String())
 
-func checkResourceInit() {
-	db, err := database.Setup()
 	if err != nil {
-		log.Errorf("error setting up db: %s", err)
+		log.Errorf("Unable to get communication %v. Err: %v", mail.PendingRevokationMember, err)
 		return
 	}
 
+	pastDueAccounts := db.GetPastDueAccounts()
+	for _, a := range pastDueAccounts {
+		if a.DaysSinceLastPayment > memberGracePeriod {
+			mailer.SendCommunication(mail.AccessRevokedLeadership, "info@hackrva.org", a)
```

Comment:
Wouldn't it be better to use the config `AdminEmail` instead of hardcoding `info@hackrva.org`?

Reply:
Updatd